### PR TITLE
Security-analysis-results: children pagination + required default sort + required sort way

### PR DESCRIPTION
**Pull request by ghelloumi**

### CODE REVIEW

File: src/components/custom-aggrid/custom-aggrid-header.js [hunk 1]
```diff
@@ -129,17 +129,17 @@ const CustomHeaderComponent = ({
...
 
     const handleSortChange = useCallback(() => {
-        let newSort = null;
-        if (!isColumnSorted || !sortWay) {
+        let newSort;
+        if (sortWay < 0) {
             newSort = 1;
-        } else if (sortWay > 0) {
+        } else {
             newSort = -1;
```
Comment (archendel) [inline src/components/custom-aggrid/custom-aggrid-header.js:136]: replace 1 and -1 by SORT_WAYS values

File: src/components/custom-aggrid/custom-aggrid-header.js [hunk 2]
```diff
@@ -209,7 +209,7 @@ const CustomHeaderComponent = ({
                         </Grid>
                         {isSortable && (
                             <Grid item>
-                                {isColumnSorted && sortWay && (
+                                {isColumnSorted && (
                                     <Grid item>
                                         <IconButton>
                                             {sortWay === 1 ? (
```
Comment (archendel) [inline src/components/custom-aggrid/custom-aggrid-header.js:215]: replace 1 with SORT_WAYS value

File: src/hooks/use-aggrid-sort.ts [hunk 3]
```diff
@@ -51,16 +49,16 @@ const getSortConfig = (
...
+    initSortConfig: ISortConfig;
 }
 
 export const useAgGridSort = ({
     dataKeyToSortKey,
     initSortConfig,
-}: IUseAgGridSortProps = {}) => {
+}: IUseAgGridSortProps) => {
     const { colKey: initColKey, sortWay: initSortWay } = initSortConfig || {
         colKey: '',
```
Comment (archendel) [inline src/hooks/use-aggrid-sort.ts:60]:
```suggestion
    const { colKey: initColKey, sortWay: initSortWay } = initSortConfig;
```

initSortConfig is mandatory, we don't need to define a default value

**Review by archendel — CHANGES_REQUESTED** [review]
- default order should be ASC, not DESC
- everytime we switch tabs, we are fetching enums, I think Anis already dealt with this bug
- limitType and side can't be filtered in "Constraint to Contingencies" view
- enum values are not translated, which makes filters a bit weird in french

File: src/components/custom-aggrid/custom-aggrid-header.js (hunk 1 above)
Comment (ghelloumi) [reply src/components/custom-aggrid/custom-aggrid-header.js]: done, thnak you

File: src/components/custom-aggrid/custom-aggrid-header.js (hunk 2 above)
Comment (ghelloumi) [reply src/components/custom-aggrid/custom-aggrid-header.js]: done, thnak you

File: src/hooks/use-aggrid-sort.ts (hunk 3 above)
Comment (ghelloumi) [reply src/hooks/use-aggrid-sort.ts]: done, thnak you

**Review by archendel — APPROVED** [review]

**Review by anistouri — APPROVED** [review]

**Pull request merged**